app/views/template.scala.html

```diff
@@ -240,6 +240,18 @@
 
     <!-- main content -->
     <div class="container main-content">
+
+    @conference.map { conf =>
+        @if(conf.infoTexts != null) {
+            @for(info <- conf.infoTexts) {
+                @if(info.indexOf("notice") == 0 && info.split("notice: ").length > 0) {
+                    <div class="jumbo-small">
```

Comment:
The current version looks very plain. Maybe just use `jumbotron` instead of `jumbo-small` for now and we can come up with a specific notice jumbotron later.